docs/monthly-meeting/2023-12.md

```diff
@@ -0,0 +1,79 @@
+- Petr Viktorin / `@encukou`
+- Daniele Procida / `@danieleprocida`
+- Hugo van Kemenade / `@hugovk`
+- Jim DeLaHunt / `@JDLH`
+- Ryan Duve / `@ryan-duve`
+- Ned Batchelder / `@nedbat`
+- Bradley / `@shenanigansd`
+- Ezio Melotti / `@ezio-melotti`
+- CAM Gerlach / `@CAM-Gerlach`
+- Usman Akinyemi / `@Unique-Usman`
+
+## Discussion
+
+* [Ryan] standard library documentation reference and adding types to it in a structured way.  For example, `collections.Counter.most_common` could be written
+
+      most_common(n: int) -> list[tuple[Any, int]]
+
+  [Petr] ... or `list[tuple[KeyType, int]]`?
+
+  - this has been discussed. There was opposition to adding the types to the stdlib code itself.
+  - [JDLH] I sometimes wish for more normative, less chatty module documentation. e.g. [*heapq* module](https://docs.python.org/3/library/heapq.html). e.g. old issue 29428 [*Doctest documentation unclear about multi-line fixtures*](https://bugs.python.org/issue29428).
+
+* [Petr] Use emoji to illustrate good and bad example commit messages
+  * https://devguide.python.org/getting-started/git-boot-camp/#accepting-and-merging-a-pull-request
+  * [python/devguide#1235](https://github.com/python/devguide/pull/1235)
+  * Should we do this for PEPs? see [python/docs-community#22](https://github.com/python/docs-community/issues/22)
+  * [Hugo] See also [python/peps#3567](https://github.com/python/peps/pull/3567) for green/red sidebars for good/bad example code in PEPs
+  * [JDLH] don't forget accessibility constraints when coming up with the style guide. e.g. some readers are red/green colorblind. ✅❌ are good in that they are legible even without color.
+
+* [Ryan] Thoughts about completing TOML->JSON conversion table: https://docs.python.org/3/library/tomllib.html#conversion-table and the TOML spec: https://toml.io/en/v1.0.0#spec
+
+
+## Reports and celebrations
+
+- PEP 732 ("The Python Documentation Editorial Board") has been submitted to the steering council [python/steering-council#220](https://github.com/python/steering-council/issues/220)
+
+- [Jim] FYI, Unicode Standard is changing form of authoritative standard documents from PDF to HTML, with corresponding change to document production tooling. If this is interesting I can provide more info. I am in the working group which is working on the new tooling.
+
+## Follow-ups from previous meeting(s)
+
+* [Petr] [Railroad diagrams](https://discuss.python.org/t/36709/20)
+  * Streams have kinda been on other topics too
+
+* Analytics (Plausible) - CAM sent an e-mail, no reply yet
+    * CAM sent multiple follow-ups
+    * Hugo sent a follow-up two weeks ago
+    * Discussion on core dev Discord that is supportive
+    * Still no reply... :(
+
+* As a beginner, what's a good starting point?
+  * There are some "easy" issues, although they can occasionally not be actually-easy.
+  * Interested in documentation, look at Diátaxis and make changes?
+    * Diátaxis is kinda abstract for newcomers.
+
```

Comment:
```suggestion
```

Let's remove this if it wasn't discussed again in the meeting. (Checking the previous month's meeting notes, seems like there is no further addition to this one)